frontend/src/scenes/error-tracking/queries.ts

```diff
@@ -155,11 +155,9 @@ export const errorTrackingGroupEventsQuery = ({
     const group = filterGroup.values[0] as UniversalFiltersGroup
     const properties = group.values as AnyPropertyFilter[]
 
-    const where = [
-        `has(${stringifyFingerprints(
-            fingerprints
-        )}, JSONExtract(ifNull(properties.$exception_fingerprint,'[]'),'Array(String)'))`,
-    ]
+    // TODO: fix this where clause. It does not take into account the events
+    // associated with issues that have been merged into this primary issue
+    const where = [`eq(${issueId}, properties.$exception_issue_id`]
```

Comment:
Does this need a trailing `)`?

Reply:
Absolutely!

Reply:
Good point. Will look and see how that events query works. Might shed some light on how they do the lookup